app/controllers/thread_controller.php

```diff
@@ -0,0 +1,76 @@
+<?php 
+class ThreadController extends AppController            
+{
+    public function index()                        
+    {
+        $threads_per_page = 15; 
+        $pageOne = 1;
+        $page = Param::get('page',$pageOne);
+        $pagination = new SimplePagination($page, $threads_per_page);
+        $threads = Thread::getAll($pagination->start_index - 1, $pagination->count + 1);
+        $pagination->checkLastPage($threads);
+        $total = Thread::countAll();
+        $pages = ceil($total / $threads_per_page);
+        $this->set(get_defined_vars());
+    }   
+
+    public function view()                
+    {
+        $thread = Thread::get(Param::get('thread_id'));
+        $comments = $thread->getComments();
+        $this->set(get_defined_vars());
+    }
+
+    public function write()                        
```

Comment:
I think this is a comment function? Why not put it on a different controller. :bow: